Framework/Geometry/test/IndexingUtilsTest.h

```diff
@@ -372,8 +372,8 @@ class IndexingUtilsTest : public CxxTest::TestSuite {
                             {2.66668320, 5.29605670, 7.9653444}};
 
     std::vector<V3D> directions;
-    for (size_t i = 0; i < 5; i++)
-      directions.emplace_back(vectors[i][0], vectors[i][1], vectors[i][2]);
+    for (auto &vector : vectors)
```

Comment:
`vector` seems like a risky variable name - could this be something like `direction` instead? The same goes for line 413